docs/tutorials/setting-up-an-nfs-fileShare.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Setting Up an NFS Fileshare
+subject: Networking
+author: Anton Vanhoucke, Ralph Hempel
+---
+
+* Table of Contents
+{:toc}
+
+## Why Do It?
+
+The nfs protocol is the standard Linux way to share files between computers, and since you're running Linux on the EV3, and there's nfs support built-in, you can share files between your computer and the EV3.
+
+Using a file share method like nfs makes it easier for folks like me to load and unload modules that are under development. That saves time and is less error prone than swapping SD Cards and rebooting all the time.
+
+For general users, it makes it easy to upgrade their SD Cards with known good modules, new kernel `uImage` files, and other updates to config files. It also makes it easier to backup your SD card. Finally, it allows you to edit files on your host machine, then run them on the EV3 without having to actually copy files anywhere!
```

Comment:
I think that the "edit files then run them" bit is the most important for the majority of users, so it might make sense to lead with that.